packages/@o3r/testing/project.json

```diff
@@ -46,7 +46,10 @@
       "options": {
         "script": "build:schematics"
       },
-      "inputs": ["schematics", "^source"],
+      "inputs": [
```

Comment:
why isn't it the default one from the nx.json ?
same for output just below ?
(I know it's not from you)

Reply:
Why it's not aligned with other packages then ?